src/Redux/actionCreators/ServiceActions.js

```diff
@@ -43,7 +44,7 @@ export const fetchServiceSuccess = res => dispatch => {
 
 export const fetchService = (pagination, filters = []) => async dispatch => {
   dispatch(loaderActions.startAIServiceListLoader);
-  let url = new URL(`${APIEndpoints.GET_SERVICE_LIST.endpoint}/service`);
+  let url = new URL(`${APIEndpoints.CONTRACT.endpoint}/service`);
```

Comment:
Can be const?

Reply:
resolved in [commit](https://github.com/singnet/refactored-dapp/pull/109/commits/b5100b7367d4ad7de42f0c97f844a6ad682f316a)